CONTRIBUTING.md

```diff
@@ -154,6 +154,69 @@ pip install -r requirements-devel.txt
 and you will need to install recent git-annex using appropriate for your
 OS means (for Debian/Ubuntu, once again, just use NeuroDebian).
 
+Contributor Files History
+-------------------------
+
+The original repository provided a [.zenodo.json](.zenodo.json)
+file, and we generate a [.contributors file](.all-contributorsrc) from that via:
+
+```bash
+pip install tributors
+tributors --version
+0.0.18
+```
+
+If you want to have Orcid identifiers, you need to export the following environment
+variables. It also helps to have a GitHub token to increase API limits:
+
+```bash
+export ORCID_ID=xxxxxxxxxx
+export ORCID_SECRET=xxxxxxxxxxxxxxxxxx
+export ORCID_TOKEN=xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx
+export ORCID_REFRESH_TOKEN=xxxxxxxxxxxxxxxxxxxxxxxxx
+export GITHUB_TOKEN=xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx
+```
+
+Instructions for these environment variables can be found [here](https://con.github.io/tributors/docs/getting-started#2-environment). 
+Then update zenodo:
+
+```bash
+tributors update  zenodo
+INFO:    zenodo:Updating .zenodo.json
+INFO:    zenodo:Updating .tributors cache from .zenodo.json
+WARNING:tributors:zenodo does not support updating from names.
+```
+
+In the case that there is more than one orcid found for a user, you will be given a list
+to check. Others will be updated in the file. You can then curate the file as you see fit.
+We next want to add the .allcontributors file:
+
+```bash
+$ tributors init allcontrib
+INFO:allcontrib:Generating .all-contributorsrc for datalad/datalad
+$ tributors update allcontrib
+INFO:allcontrib:Updating .all-contributorsrc
+INFO:allcontrib:Updating .tributors cache from .all-contributorsrc
+INFO:allcontrib:⭐️ Found new contributor glalteva in .all-contributorsrc
+INFO:allcontrib:⭐️ Found new contributor adswa in .all-contributorsrc
+INFO:allcontrib:⭐️ Found new contributor chrhaeusler in .all-contributorsrc
+...
+INFO:allcontrib:⭐️ Found new contributor bpoldrack in .all-contributorsrc
+INFO:allcontrib:⭐️ Found new contributor yetanothertestuser in .all-contributorsrc
+INFO:allcontrib:⭐️ Found new contributor bhanuprasad14 in .all-contributorsrc
+WARNING:tributors:allcontrib does not support updating from orcids.
+WARNING:tributors:allcontrib does not support updating from email.
+```
+
+We can then populate the shared .tributors file:
+
+```bash
+$ tributors update-lookup allcontrib
+```
+
+And then we can rely on the [GitHub action](.github/workflows/update-contributors.yml) to update contributors, and
+look for new orcids as we did above.
```

Comment:
Please clarify that it would send a PR, correct?

Reply:
Yes, when a contribution is made (as in, _merged_ into master) it would be triggered and open a PR.

Reply:
so should be reflected in the text above?